tools/adventure-pack/goodies/java/src/union_find/UnionFind.java

```diff
@@ -26,7 +44,7 @@ public int find(int a) {
     return parent[a] = find(parent[a]);
   }
 
-  public boolean areConnected(int a, int b) {
-    return find(a) == find(b);
-  }
+  public boolean areConnected(int a, int b) { return find(a) == find(b); }
```

Comment:
The other thing that's failing is the formatting -- I believe the Java Prettier plugin would not one-line this, or the other function below. Run `yarn format` anywhere in the `tools` directly to have Prettier reformat this.

Reply:
Done.